releasecalendar/data.go

```diff
@@ -0,0 +1,49 @@
+package releasecalendar
+
+// Release represents a release
+type Release struct {
+	URI                       string              `json:"uri"`
```

Comment:
Generally I would order the fields alphabetically, it is easier to find the field you are looking for that way.

The only reason not to do it alphabetically is if there is a logical order, e.g. grouping fields together. Here I think alphabetically works best